packages/translations/src/i18next/i18next.ts

```diff
@@ -167,6 +167,12 @@ export const localize = (string: string, values?: Record<string, unknown>) => {
     return i18n.t(crc32(string).toString(), { defaultValue: string, ...values });
 };
 
+export const useLocalize = () => {
+    useTranslation();
```

Comment:
why are we just invoking the hook without using anything from `useTranslation`?

Reply:
@shahzaib-deriv Because just calling the `useTranslation` hook re-trigger the current hook and as result we trigger to repaint the component which uses `localize` function from the hook